models/comments_models.js

```diff
@@ -8,4 +8,20 @@ exports.selectCommentsByArticleId = async (article_id) => {
         [article_id]
     )
     return result.rows
+}
+exports.insertComment = async (body, author, article_id) => {
+    const result = await db.query(
+        `INSERT INTO comments
+        (body, author, article_id)
+        VALUES
+        ($1, $2, $3)
+        RETURNING *;`,
+        [body, author, article_id]
+    )
+    if (result.rows.length === 0) {
+        return Promise.reject({ status: 400, msg: 'Bad request: comment posting failed' })
+    }
```

Comment:
Feel like this if statement is unnecessary. There would be no empty rows array as if there's an error, SQL will ensure the promise has rejected